README.md

```diff
@@ -1,27 +1,73 @@
 # sdl_sublime_log_plugin
 Sublime Text plugin  for handy work with SdlLogs.
 
-## Requirements
+## Installation
+
+### Requirements
 This plugin requires at least Sublime Text 3 v.3176 or higher for best performance and correct syntax
 
-## Installing
-- Move directory "SdlLogs" to Sublime Packages folder.
+- [yaml](https://stackoverflow.com/questions/14261614/how-do-i-install-the-yaml-package-for-python)
+- [pip](https://pip.pypa.io/en/stable/installing/)
+
+### How to install sdl_sublime_log_plugin:
+1. Clone or download this repository. 
+2. Copy "SdlLogs" folder to Sublime Packages folder.
 (Generally at /home/*USER*/.config/sublime-text-3/Packages)
-- Change syntax in Sublime to SdlLogs.
+3. Change syntax in Sublime to SdlLogs.
 (View -> Syntax ->SdlLogs)
 
-## Dependencies
-- Should install package for yaml
-(Enter in terminal 'sudo apt-get install python-yaml')
+## Working with sdl_sublime_log_plugin
 
-## Functionality
-This plugin implements color scheme and simple commands to work with SdlLogs
-- Indicates methods and make them clickable (Opens file containing that method and highlight him)
-- Search for common syntax patterns and highlight them (color scheme)
-- Implements commands for filtering columns&strings, finding not closed functions
+### Configuration
 
-## Configuration
 Open Sublime in top panel choose preferences->Package->Settings->SdlLogs:
-- Item "Color Scheme" contain configuration for text color.
-- Item "Setting" contain all path to configure files and path to your sources if logs exists incorrect absolute path to file.
-- Item "Key Bindings" contain all keybindings configuration.
+
+1. Item "Color Scheme" contain configuration for text color.
+2. Item "Setting":
+    - `"color_scheme"` path to color configuration file.
+    - `"syntax"` path to syntax configuration file.
+    - `"extensions"` contain file extensions with which the plugin works.
+    - `"source_path"` path to your source code directory (need for work recursive search)
+    - `"check_keys_override"` responsible flag for turn on/off 'Key bindings override' function.
+3. Item "Key Bindings" contain all keybindings configuration.
+
+### Jump to file
+Move focus to the line which contains path, then you can press `"F12"` (by default) after this, you jump to file and focus to definition line.
+
+### Key bindings override
+After each starting sublime, plugin check key bindings override with other plugin, default and user key bindings. 
+If such exist will show message with problem key bindings and open key binding settings. You can turn off this function change `check_keys_override` tag value in setting file to false.
```

Comment:
@dkandyba Will be better like as:
After each start sublime application, plugin will checks  installed plugins (if them exists), default and user key bindings for override conflicts.
If conflicts will be found, plugin will opens separate tab with all conflicts and their descriptions, and will opens separate window with key bindings settings for changing them.
You can turn off this function change `check_keys_override` tag value in setting file to **false**.